### DISCUSSION TITLE
Welcome to Hass.io-custom-component-spaclient Discussions!

### DISCUSSION THREAD

**Discussion by plmilord** [plmilord/Hass.io-custom-component-spaclient#16] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Jieper001** [comment]
How can I buy you a coffee?  (not joking, I am using the integration virtually every day and is the reason why I got started with HA as the BWA app is utter cr@p).

Great stuff!!
